### PR TITLE
PAF-1: Create skeleton app

### PR DESCRIPTION
**Pull request by Rhodine-orleans-lindsay**
## What 
- Created initial skeleton app for paf, with initial routes

### CODE REVIEW

File: assets/js/index.js [hunk 1]
```diff
@@ -0,0 +1,13 @@
...
+
+var $ = require('jquery');
+
+var accessibleAutocomplete = require('accessible-autocomplete');
+$('.typeahead').each(function applyTypeahead() {
+  accessibleAutocomplete.enhanceSelectElement({
+    defaultValue: '',
+    selectElement: this
+  });
+})
```
Comment (sulthan-ahmed) [inline assets/js/index.js:13]: if you're not using, I wouldn't add it.  It makes sense to add it as part of a PR if I'm honest.

File: config.js [hunk 2]
```diff
@@ -0,0 +1,20 @@
...
+  },
+  useMocks: useMocks,
+  upload: {
+    maxFileSize: '10mb',
+    // if mocks set use file service served up by app otherwise use filevault's port 3000
+    hostname: !useMocks && process.env.FILE_VAULT_URL ?
+      process.env.FILE_VAULT_URL :
+      `http://localhost:${useMocks ? (process.env.PORT || 8080) : 3000}/file`
+  }
+};
```
Comment (sulthan-ahmed) [inline config.js:20]: again, I would avoid adding things that are not really related to your PR.

When you actually work on filevault add it then.

You really want a barebones PR

when someone goes back in history it makes it harder to understand otherwise

**Review by sulthan-ahmed — COMMENTED** [review]
good stuff, just some minor things

File: apps/paf/fields/index.js [hunk 3]
```diff
@@ -0,0 +1,6 @@
+'use strict';
+
+
+module.exports = {
+
+};
```
[the remark this answers is not in the document]
Comment (Rhodine-orleans-lindsay) [reply apps/paf/fields/index.js]: have removed the code from this now

File: apps/paf/translations/src/en/fields.json [hunk 4]
```diff
@@ -0,0 +1 @@
+{}
```
[the remark this answers is not in the document]
Comment (Rhodine-orleans-lindsay) [reply apps/paf/translations/src/en/fields.json:1]: this does not work empty (the app does not build), so I have left this as is.

File: assets/js/index.js (hunk 1 above)
Comment (Rhodine-orleans-lindsay) [reply assets/js/index.js]: I have removed this now

File: config.js (hunk 2 above)
Comment (Rhodine-orleans-lindsay) [reply config.js:8]: I have removed this for now

File: apps/paf/translations/src/en/fields.json (hunk 4 above)
[the remark this answers is not in the document]
Comment (sulthan-ahmed) [reply apps/paf/translations/src/en/fields.json:1]: ok, fair enough

File: config.js [hunk 5]
```diff
@@ -0,0 +1,8 @@
+'use strict';
+/* eslint no-process-env: 0 */
+
+const env = process.env.NODE_ENV || 'production';
```
Comment (sulthan-ahmed) [inline config.js:4]: I would remove defaults because of the problems it can cause and confusion

**Review by sulthan-ahmed — APPROVED** [review]
lgtm
